cockatrice/src/shortcutssettings.h

```diff
@@ -126,359 +126,407 @@ class ShortcutsSettings : public QObject
-        {"Player/aCreateRelatedTokens", SK(QT_TRANSLATE_NOOP("shortcutsTab", "Create all related tokens"),
-                                           parseSequenceString("Ctrl+Shift+T"),
-                                           ShortcutGroup::Playing_Area)},
-        {"Player/aCreateAnotherToken", SK(QT_TRANSLATE_NOOP("shortcutsTab", "Create another token"),
-                                          parseSequenceString("Ctrl+G"),
-                                          ShortcutGroup::Playing_Area)},
-        {"Player/aSetAnnotation",
-         SK(QT_TRANSLATE_NOOP("shortcutsTab", "Set annotation"), parseSequenceString(""), ShortcutGroup::Playing_Area)},
-        {"Player/aMoveToBottomLibrary", SK(QT_TRANSLATE_NOOP("shortcutsTab", "Bottom library"),
-                                           parseSequenceString(""),
-                                           ShortcutGroup::Move_selected)},
-        {"Player/aMoveToExile",
-         SK(QT_TRANSLATE_NOOP("shortcutsTab", "Exile"), parseSequenceString(""), ShortcutGroup::Move_selected)},
-        {"Player/aMoveToGraveyard", SK(QT_TRANSLATE_NOOP("shortcutsTab", "Graveyard"),
-                                       parseSequenceString("Ctrl+Del"),
-                                       ShortcutGroup::Move_selected)},
+         ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "End"), parseSequenceString("F10"), ShortcutGroup::Game_Phases)},
+        {"Player/aNextPhase", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Next phase"),
+                                          parseSequenceString("Ctrl+Space;Tab"),
+                                          ShortcutGroup::Game_Phases)},
+        {"Player/aNextPhaseAction", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Next phase action"),
+                                                parseSequenceString("Shift+Tab"),
+                                                ShortcutGroup::Game_Phases)},
+        {"Player/aNextTurn", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Next turn"),
+                                         parseSequenceString("Ctrl+Return;Ctrl+Enter"),
+                                         ShortcutGroup::Game_Phases)},
+        {"Player/aTap", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Tap / Untap Card"),
+                                    parseSequenceString(""),
+                                    ShortcutGroup::Playing_Area)},
+        {"Player/aUntapAll", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Untap all"),
+                                         parseSequenceString("Ctrl+U"),
+                                         ShortcutGroup::Playing_Area)},
+        {"Player/aDoesntUntap", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Toggle untap"),
+                                            parseSequenceString(""),
+                                            ShortcutGroup::Playing_Area)},
+        {"Player/aFlip", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Turn card over"),
```

Comment:
Should we rename the code here too? `aFlip` --> `aTurnOver`

At one point we hopefully get a real flip action as well to rotate cards 180° to properly play with Kamigawa flip cards or other effects from different games which require that.

Reply:
no that would break people's settings